cadctl/cmd/root.go

```diff
@@ -41,4 +45,22 @@ func Execute() {
 
 func init() {
 	rootCmd.AddCommand(investigate.InvestigateCmd)
+	rootCmd.PersistentFlags().StringP("ocmconfig", "c", "", "Path to the OCM config file to use for CLI requests. When not set the OCM_CONFIG environment variable will be used. If that is not set, the default OCM config locations will be file will be used.")
+}
+
+func loadCommon(cmd *cobra.Command, args []string) error {
+	var err error
+
+	_, err = config.Config.GetLogLevel(cmd)
+	if err != nil {
+		logging.Fatal(err)
+		return err
```

Comment:
Doesn't fatal already return? https://pkg.go.dev/go.uber.org/zap#Logger.Fatal 

We can re-write this without the error passing and fatal only instead.

Reply:
I removed this part of code.